ENIGMAsystem/SHELL/Platforms/SDL/Joystick.cpp

```diff
@@ -0,0 +1,126 @@
+#include "SDL2/SDL.h"
```

Comment:
You also need to add a copyright header here and to the top of `Joystick.h` with your name. Copy and paste the one from `WINDOWSjoystick.cpp` then replace `2013-2014 Robert B. Colton` with `2018 Samuel Venable`.

```cpp
/** Copyright (C) 2018 Samuel Venable
***
*** This file is a part of the ENIGMA Development Environment.
***
*** ENIGMA is free software: you can redistribute it and/or modify it under the
*** terms of the GNU General Public License as published by the Free Software
*** Foundation, version 3 of the license or any later version.
***
*** This application and its source code is distributed AS-IS, WITHOUT ANY
*** WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
*** FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
*** details.
***
*** You should have received a copy of the GNU General Public License along
*** with this code. If not, see <http://www.gnu.org/licenses/>
**/
```